common/components/WalletDecrypt/components/DeterministicWalletsModal.tsx

```diff
@@ -113,21 +122,20 @@ class DeterministicWalletsModalClass extends React.Component<Props, State> {
         handleClose={onCancel}
       >
         <div className="DWModal">
+          {/* TODO: replace styles for flexbox with flexbox classes in https://github.com/MyEtherWallet/MyEtherWallet/pull/850/files#diff-2150778b9391533fec7b8afd060c7672 */}
           <form className="DWModal-path form-group-sm" onSubmit={this.handleSubmitCustomPath}>
-            <span className="DWModal-path-label">Addresses for</span>
-            <select
-              className="form-control"
+            <span className="DWModal-path-label">Addresses </span>
+            <Select
+              name="fieldDPath"
+              className=""
+              value={this.state.currentLabel || this.findDPath('value', dPath).value}
```

Comment:
I feel like the nomenclature of using "label" for something that's a value is fairly confusing.

Reply:
Agreed, however these were modifications, not additions. Ideally DeterministicWalletsModal could be re-written to use much less component state, and these types of changes could be made then.